salt/metalk8s/calico/configured.sls

```diff
@@ -0,0 +1,56 @@
+{%- from "metalk8s/map.jinja" import kube_api with context %}
+{%- from "metalk8s/map.jinja" import defaults with context %}
+
+{%- set ca_server = salt['mine.get']('*', 'kubernetes_ca_server').keys() %}
+
+{%- if ca_server %}
+
+Create kubeconf file for calico:
+  metalk8s_kubeconfig.managed:
+    - name: /etc/kubernetes/calico.conf
+    - ca_server: {{ ca_server[0] }}
+    - signing_policy: {{ kube_api.cert.client_signing_policy }}
+    - client_cert_info:
+        CN: {{ salt['network.get_hostname']() }}
+        O: metalk8s:calico-node
+    - apiserver: https://{{ kube_api.service_ip }}:443
+    - cluster: {{ defaults.cluster }}
+
+Create CNI calico configuration file:
+  file.serialize:
+    - name: /etc/cni/net.d/10-calico.conflist
+    - user: root
+    - group: root
+    - mode: 664
```

Comment:
`644`?

Reply:
Calico default it to 664 :p but will change